plugins/extractit/icon_extractit/util/extractor.py

```diff
@@ -138,11 +138,12 @@ def clear_domains(matches: list) -> list:
         # Here we remove the path, this solution allows to prevent the situation where the file names from the URL will
         # be extracted as domains, e.g. www.example.com/test.html
         split_match = match[1].split("/")[0]
+        split_match = split_match.split(":")[0]
```

Comment:
```suggestion
        # Here we remove the port number, if present, which could prevent a legitimate domain name match.
        # Ex. ssh.example.com:22 becomes ssh.example.com
        split_match = split_match.split(":")[0]
```

Reply:
Is there a point earlier in the code where protocol prefixes are removed?
If we pass in to this method a string such as `https://example.com:443` that would break the assumption we make in this code line that the first instance of `/` or `:` delimit the port and path.

Reply:
Protocol prefixes are not included in matches for domains, so there is no need to remove them.